task/sinker.go

```diff
@@ -302,7 +304,7 @@ func (s *Sinker) applyConfig(newCfg *config.Config) (err error) {
 }
 
 func (s *Sinker) applyFirstConfig(newCfg *config.Config) (err error) {
-	util.Logger.Info("going to apply the first config", zap.Reflect("config", newCfg))
+	util.Logger.Info("going to apply the first config")
```

Comment:
Log config helps debug. The better way is to customize JSON serialization of `Config` structure to mask credentials.

Reply:
Instead of custom zap encoder implementation I decided to simply omit passwords in json encoding. LMK if that works for you

Reply:
That's fine for me. Go on!

Reply:
Okay. It doesn't look easy and fast to implement arbitrary fields obfuscation in zap. I'd like to take time and think about it, eventually implementing in a separate PR. Will it work for you, if I revert the logging part completely?